backend/entrypoint

```diff
@@ -17,10 +17,14 @@ start_backend() {
     supervisord -n
 }
 
-if [ "$version" == "$DATABASE_VERSION" ]; then
+if [ $version -eq $DATABASE_VERSION ]; then
+    echo "[+] Running on intended database version ($DATABASE_VERSION).. No migration required"
+    start_backend
+elif [ $version -gt $DATABASE_VERSION ]; then
```

Comment:
@slowr till now the migration was called, but did not do anything, effectively allowing the user to continue. This PR simply formalizes this behavior and does not trigger unnecessarily migration. Do you think we should stop execution if the intended version is older than the current one? (a user e.g., may see info like new hijack types without being able to manage them at the containers). How can we avoid this?

Reply:
The problem is that in that case we should upgrade the containers rather than the DB (since we cannot go back). Maybe we can instruct the user to set the SYSTEM_VERSION to "latest" and exit?

Reply:
having the tag latest is not enough because you also need to docker pull the latest image each time

Reply:
@slowr please check the updated behavior (advising git and docker-compose pull and then exiting), and if you agree, approve this PR to be included in the coming patch release.